src/main/java/school/lemon/changerequest/java/introduction/hw1/Task10.java

```diff
@@ -3,7 +3,24 @@
 
 public class Task10 {
 
-    public static boolean containDigitTwo(int n) {
-        return false;
+    public static boolean containDigitTwo(int n)
```

Comment:
Please format your code. For IntelliJ you can do this via 'Code -> Reformat Code' or just Ctrl + Alt + L. And do it for all tasks.